package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "@snowplow/react-native-tracker",
-  "version": "0.1.7",
+  "version": "0.2.0",
   "description": "A library for tracking Snowplow events in React Native",
-  "main": "index.js",
+  "homepage": "https://docs.snowplowanalytics.com/docs/collecting-data/collecting-from-own-applications/react-native-tracker/",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "scripts": {
+    "compile": "rm -rf dist && tsc -p .",
```

Comment:
I'd suggest switching to [rimraf](https://www.npmjs.com/package/rimraf) CLI as a dev dependency to remove these files and folders in the `compile` and `build` commands as `rm -rf` is a unix command so won't work on Windows.